tests/wsl/firstrun.pm

```diff
@@ -8,17 +8,23 @@
 # without any warranty.
 
 # Summary: Configure WSL users
-# Maintainer: Martin Loviska <mloviska@suse.com>
+# Maintainer: qa-c  <qa-c@suse.de>
 
-use base "windowsbasetest";
-use strict;
-use warnings;
+use Mojo::Base qw(windowsbasetest);
 use testapi;
-use version_utils 'is_sle';
+use version_utils qw(is_sle);
+
+sub is_sut_dev {
+    return !!get_var('BETA', 0);
```

Comment:
just an idea for readability
`return get_var('BETA', 0) ? 1 : 0`
Although do you think it is a good idea to wrap this in a function instead just put a comment (assuming again that you try to make the code more explanatory here)?

Reply:
On the second thought, it makes sense just to leave plain `get_var('BETA')` call